VocaDbWeb/Views/Shared/KnockoutPartials/_CoverArtistMessage.cshtml

```diff
@@ -0,0 +1,16 @@
+﻿<!--
+	Shows a message for cover artists in an original song.
+	Binding context:  KnockoutComputed<dc.ArtistContract[]>
+-->
+<div class="alert" style="display:none" data-bind="visible: coverArtists() && coverArtists().length > 0 && !canHaveOriginalVersion()">
+	<partial name="Partials/Shared/_NotificationIcon" />
+	@ViewRes.EntryCreateStrings.FoundCoverArtist

```

Comment:
Causes the AppVeyor build to fail. Will be resolved after the changes from #817 get merged into this PR.